src/main/java/core/basesyntax/FileReaderImpl.java

```diff
@@ -0,0 +1,20 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
```

Comment:
The class `FileReaderImpl` should be in its own package, according to the checklist it should be placed under a package named `service` or `service.impl`.